### DISCUSSION TITLE
Gray Video

### DISCUSSION THREAD

**Discussion by ucaglarcaliskan** [svpcom/wfb-ng#162] [Questions]
Hi,

I am trying send and receive a video to test. But sometimes the video becomes gray and fps doesn't look good. 
I use this commands:
[Commands](https://gist.github.com/reinzor/812a1dadd62dcf2a309c9c99af92244f)

When I increase the bitrate, getting more gray and low fps. I use AR9271 chipset. Is this a problem?

This is test video: [Video](https://youtu.be/G1uJJdaCF_k)

**Comment by svpcom** [comment]
1. What bitrate do you use? In examples above x264enc bitrate didn't set
2. Try without WFB (use network adapter instead) - it may be h264 encoder/decoder issues 
3. Run wfb-cli to see packet statistics (how many lost, fec recovered, etc)
4. Don't use 2.4 GHz card. This band is very crowded in urban areas
5. Do you use latest version of WFB? There was a FEC bug in previous releases
6. Ensure that you disable NetworkManager, wpa_supplicant, dhclient on WFB wlan adapter (see HOWTO)

**Reply by svpcom** [reply]
Fow MCS#1 max recommended video bandwidth ~4 mbit/s

**Reply by ucaglarcaliskan** [reply]
Thank you for your fast reply.

1. In this video, I set bitrate as 128. When I use 2048, there are more gray frames.
2. I tried on the same pc and it worked well. But, I will try on the different pcs using network adapter.
3. I will add a screenshot about that.
4. Is 2.4 GHz enough in a noise-free environment? We can not get 5GHz adapter easily. Stocks are empty.
5. I have cloned previous week but I will update it.

Also, I can't understand "Fow MCS#1". Could you please give me more details?

**Reply by svpcom** [reply]
2.4GHz is enough in noise-free environment (but it will hard to find that)
https://aliexpress.com/wholesale?catId=&SearchText=rtl8812au
MCS#1 is QPSK 1/2 modulation, see http://mcsindex.com/

**Reply by svpcom** [reply]
I use https://aliexpress.com/item/4000542389424.html, but it out of stock now

**Comment by didadidadi5** [comment]
Do you use latest version of WFB? There was a FEC bug in previous releases
这个bug确实会引起间歇性灰图。
